### PR TITLE
docs: improve api docs for guidance

### PR DESCRIPTION
**Pull request by allencloud**
Signed-off-by: Allen Sun <allensun.shl@alibaba-inc.com>

**1.Describe what this PR did**

improve api docs for guidance:
First, I list three ways to interact with pouchd, cli, API and sdk;
Second, describe what is this directory's usage in details.

**2.Does this pull request fix one issue?** 

NONE

**3.Describe how you did it**
add more specific guidance in Pouch API doc.

**4.Describe how to verify it**
NONE

**5.Special notes for reviews**
/cc @Letty5411

### CODE REVIEW

File: apis/README.md [hunk 1]
```diff
@@ -1,16 +1,22 @@
-# API
+# Pouch API
 
-Directory /apis currently contains the following three kinds of things:
+We encourage pouch users to experience Pouch in different ways. When users do this, there are three ways we could choose to interact with Pouchd mostly:
```
Comment (Letty5411) [inline apis/README.md]: we could -> they could or you could?

File: apis/README.md [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
 
-* API definitions via [swagger.yml](swagger.yml);
-* struct in `/apis/types` used in restful API between Pouch Client and Server, both auto generated and manual added;
-* api server implemented for pouch daemon.
+* For end-users, command line tool `pouch` is mostly used.
+* For developer, pouchd's raw API is the original thing they would make use of. For more details about API docs, please refer to [API.md](../docs/api/API.md). We should also keep it in mind that doc [API.md](../docs/api/API.md) is automatically generated by [swagger2markup](https://github.com/Swagger2Markup/swagger2markup). Please **DO NOT** edit [API.md](../docs/api/API.md) directly.
```
Comment (Letty5411) [inline apis/README.md]: developer -> developers

File: apis/README.md [hunk 3]
```diff
@@ -20,13 +26,13 @@ Swagger is a simple yet powerful representation of your RESTful API. [go-swagger
 
 ### Design API 
 
-With swagger installed in you local $PATH, you could start to design or update API of Pouch. Here we need to underline that every change for API should first in [swagger.yml](swagger.yml) of Pouch porject.
+With swagger installed in you local $PATH, you could start to design or update API of Pouch. Here we need to underline that every change for API should be firstly added in [swagger.yml](swagger.yml) of Pouch porject.
```
Comment (Letty5411) [inline apis/README.md]: you local -> your local

File: apis/README.md (hunk 2 above)
Comment (Letty5411) [reply apis/README.md]: [API.md](../docs/api/API.md)  is not found.

**Comment by allencloud** [comment]
Thanks for the review. Updated now. @Letty5411

**Comment by Letty5411** [comment]
@allencloud  LGTM

**Pull request merged**